content/rs/administering/cluster-operations/maintenance-mode.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: Maintenance Mode for Cluster Nodes
+description: Prepare node for maintenance, and restore it to previous state
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+From time to time, a node requires maintenance, for reasons relating to
+your organization, not to the Redis Enterprise cluster. The Maintenance
+Mode is a process to simplify the process of moving Redis shards away
+from node, and returning them after the maintenance is done.
+
+## Basic Usage
+
+Connect via SSH to the cluster, and use rladmin:
+
+```src
+$ rladmin node <id> maintenance_mode on
+```
+
+Wait for cluster to remove all Redis shards from node, and perform
+maintenance. Once maintenance is done, use the following to return the
+shards that were on the node back:
+
+```src
+$ rladmin node <id> maintenance_mode off
+```
+
+## Maintenance On Steps
+
+1. Checks that node with that id exists in the cluster.
+1. Checks that cluster will not lose quorum if node is shut off.
+1. Cluster takes a snapshot of node. This is a record of which shards
+and endpoints are on node at that time. This is not an RDB snapshot of
+the actual data, but just the node configuration
+1. Prevents shards from migrating into the node, by making it a quorum
+node.
+1. Cluster tries to evict node. By default, all shards will be migrated
+to other nodes, and endpoints will be bound to other nodes.
+
+**Note:**
+If maintained node if the cluster's master node, process won't demote
+it. If node is restarted, the cluster will automatically elect another
```

Comment:
@RoeyPrat Does this mean that someone who wants to do maintenance on a master node should restart in order that another node will be elected master?

Reply:
No. The user doesn't have to worry about master election, it will happen when and if it has to.